tests/models/nn/transformers/test_sasrec.py

```diff
@@ -981,12 +1024,13 @@ def get_reco(model: SASRecModel) -> pd.DataFrame:
 
         model_1 = model.from_config(initial_config)
         reco_1 = get_reco(model_1)
-        config_1 = model_1.get_config(simple_types=simple_types)
+        config_1 = model_1.get_config(mode="pydantic", simple_types=False)
```

Comment:
Will it raise en error with `simple_types=simple_types`? Should pass (here and below)

Reply:
I forgot removed this after "debug"
Returned to previous version because it works.